### DISCUSSION TITLE
Minimal example for CLI?

### DISCUSSION THREAD

**Discussion by CINVESTAV-LABI** [vdemichev/DiaNN#1299] [Q&A]
Hi, I'm just starting with DIANN. I am the maintainer of MASSyPup, a Linux distribution for mass spectrometry data analyses, and want to use DIANN for its use in proteomics courses and off-site data analysis workstations.
Since no DIANN GUI exists for Linux, could anyone recommend a minimal example for running DIANN on DIA .mzML files using a fast database?
Is it only
~~~
diann --min-pr-cahrge 2 --fasta-search --fasta uniprot-human.fasta --dir /home/antix/datafiles/
~~~
Or is there anything else required?
Will DIANN report errors in running the command (such as data format not centroided or missing parameters)?
By the way, a little GUI, e.g., a tkinter-based GUI, a web interface, or anything cross-platform, would be awesome for running DIANN on Linux servers.

**Comment by vdemichev** [comment]
Hi,

You can run DIA-NN GUI under Wine on Linux.

Best,
Vadim
